docs/preact-testing-library/api.mdx

```diff
@@ -113,12 +113,12 @@ fireEvent.click(buttonNode);
 
 // Fire event Option 2.
 fireEvent(
-buttonNode,
-new Event('MouseEvent', {
-  bubbles: true,
-  cancelable: true,
-  button: 0,
-});
+  buttonNode,
+  new Event('MouseEvent', {
+    bubbles: true,
+    cancelable: true,
```

Comment:
Thanks for the fix, though I would like to re-add the `button` property.

```suggestion
    cancelable: true,
    button: 0
```

`button` is a valid event property - https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button - and is also impolemented like this in fireEvent - https://github.com/testing-library/dom-testing-library/blob/main/src/event-map.js#L82.

Reply:
@pppp606 is correct. There is no `button` property on `Event`. It is implemented on `MouseEvent`.
https://codesandbox.io/s/event-property-button-p6w8xh?file=/src/index.test.js